modules/apps/journal/journal-service/src/main/resources/META-INF/custom-sql/default.xml

```diff
@@ -218,6 +218,28 @@
 				JournalArticle.id_ ASC
 		]]>
 	</sql>
+	<sql id="com.liferay.journal.service.persistence.JournalArticleFinder.findByG_A_LtDD_GtED_ST">
```

Comment:
Hi @balazssk (just for the future): it will be better to use DSLQueries instead on legacy finders. We have been trying to replace them where possible